app/components/quick-post.hbs

```diff
@@ -111,13 +111,15 @@
     {{/each}}
 
     {{#if showLoaderButton}}
-      <button
-        class='btn btn-sm btn-block btn-secondary btn-read-more'
-        {{action 'loadMore'}}
-        type='button'
-      >
-        Meer laden
-      </button>
+      <div class="d-grid">
+        <button
```

Comment:
![image](https://user-images.githubusercontent.com/7385023/226177702-f9b0151b-c31d-4b8e-851c-3f5a5f1825c9.png)
This button can use some more x padding